bigg_com.clj

```diff
@@ -0,0 +1,77 @@
+(config
+(password-field
+:name "bearerToken" 
+:label "<Provide label here>"))
+
+(default-source
+(header-params "Accept" "application/json")
+    (paging/page-number
+        :page-number-query-param-initial-value 0
+        :page-number-query-param-name "page"
+        :limit 10 
+        :limit-query-param-name "limit" 
+    )
+                    
+(auth/http-bearer)
+(http/get :base-url "https://developer.bigcommerce.com/docs/rest-management")
+(error-handler 
+(when :status 401 :action fail)
+)
+
+)
+
+
+
+(entity ALL_CHANNEL
+    (api-docs-url "https://developer.bigcommerce.com/docs/rest-management/channels#get-all-channels")
+
+    (source (http/get :url "/channels#get-all-channels")
+    (extract-path "data"))
+
+
+    (fields
+    id : id
+    icon_url
+    is_listable_from_ui
+    is_visible
+    date_created
+    external_id
+    type
+    platform
+    date_modified
+    name
+    status
+    config_meta
+
+)
+(dynamic-fields
+(flatten-fields
+(fields
+app
+
+)
+:from config_meta)
```

Comment:
app is not a field, it is a path to fetch the id which you already mentioned in below flatten. So no need to mention in here